src/manifest.json

```diff
@@ -106,6 +106,66 @@
         "default": "Ctrl+Shift+0"
       },
       "description": "__MSG_containerShortcut__"
+    },
+    "reopen_in_container_0": {
+      "suggested_key": {
+        "default": "Alt+Shift+1"
```

Comment:
It would be best to let the default empty because conflicts with other addons or the browser are too common since there isn't a lot of key combos that are still free.

Reply:
Thanks, will do